eclipse-collections-api/src/main/java/org/eclipse/collections/api/factory/list/ImmutableListFactory.java

```diff
@@ -144,4 +144,9 @@
         MutableList<T> list = (MutableList<T>) items.toSortedList(comparator);
         return list.toImmutable();
     }
+
+    /**
+     * @since 12.0
+     */
+    <T> ImmutableList<T> ofNCopiesOf(int copies, T item);
```

Comment:
We have a similar method MutableListFactory#withNValues(). I think it's awkward to have "of" in the name twice. I'm also not sure if "copies" is appropriate in the name when these are all the same item, but then again, that's true of java.util.Collections#nCopies where the name comes from.

Right now I'm leaning toward also using `withNValues` as the name but I'd like to hear opinions.